### ISSUE TITLE
questions on my own dataset

### ISSUE THREAD

**Issue by B-Betty** [tinghuiz/SfMLearner#79]
hi,@tinghuiz
I have the depth images ,but how can i get the depth ground truth in .bin files like the velodyne_points in datasets kittti？
and,I train my own dataset ,the loss is about 3 or so. while the total loss is about 1 or so when  I train the datasets kitti or odometry ,how can i change the value learning_rate or other parameters？

**Comment by tinghuiz** [comment]
Why do you want your depth ground-truth to be like velodyne points?

Is your own dataset very different from KITTI? I'm not sure what would be a correct loss value for other datasets. There are definitely hyperparameters (e.g. smoothness weight) that could be tuned to potentially improve the performance.

**Comment by B-Betty** [comment]
thanks for your reply. @tinghuiz 
the reason why I want the depth ground-truth to be like velodyne points  is that evaluation on depth CNN.  the code depth_evaluation_utils.py  needs the depth ground-truth in form of  .bin file.

the second question have been solved.

**Comment by tinghuiz** [comment]
It's probably easier to modify the depth evaluation code to handle your specific file format. You can take a look at the ground-truth depth generated by this line: https://github.com/tinghuiz/SfMLearner/blob/master/kitti_eval/eval_depth.py#L32, and format yours in the similar manner.
